ha/core/system_health/status_mapper.py

```diff
@@ -51,7 +52,8 @@ class StatusMapper:
         "repairing"   : "informational",
         "repaired"    : "informational",
         "rebalancing" : "informational",
-        "failed"      : "error"
+        "failed"      : "error",
+        "offline"     : "informational"
```

Comment:
Is this need to add as it is provisional and may not be required in code?
@ArchanaLimaye  can you please confirm?

Reply:
Offline is required if disk is marked offline for investigation.

Reply:
ok